cfg/cfg.go

```diff
@@ -126,12 +121,6 @@ func init() {
 		"",
 		"Default address of registry from which to retrieve meta.yamls when plugin FQNs do not specify a registry. Ignored unless --download-metas is set",
```

Comment:
Please update this message as well, since `--download-metas` is being removed.

Reply:
nice catch